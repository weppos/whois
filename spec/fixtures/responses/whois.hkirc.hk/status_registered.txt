 -------------------------------------------------------------------------------
 Whois server by HKIRC
 -------------------------------------------------------------------------------
 .hk top level Domain names can be registered via HKIRC-Accredited Registrars. 
 Go to https://www.hkirc.hk/content.jsp?id=280 for details. 
 -------------------------------------------------------------------------------



Domain Name:  DISCUSS.COM.HK 
Bundled Domain Name:  討論區.公司.香港 
Domain Status: Active 

Contract Version:   HKDNR latest version   

Registrar Name: Hong Kong Domain Name Registration Company Limited

Registrar Contact Information: Email: enquiry@hkdnr.hk                       Hotline: +852 2319 1313 

Reseller: 




Registrant Contact Information:

Company English Name (It should be the same as the registered/corporation name on your Business Register Certificate or relevant documents): NETWORLD TECHNOLOGY LIMITED
Company Chinese name:  網滙科技有限公司 
Address: ROOM 15-18, 16/F., DELTA HOUSE, 3 ON YIU STREET, SHATIN, N.T., HONG KONG.     
Country: HK
Email: techsupport@networld.hk 
Domain Name Commencement Date: 14-10-2003
Expiry Date: 23-09-2016 
Re-registration Status:  Complete   



Administrative Contact Information:

Given name: OSCAR    
Family name: LEUNG  
Company name: NETWORLD TECHNOLOGY LIMITED
Address: ROOM 15-18, 16/F., DELTA HOUSE, 3 ON YIU STREET, SHATIN, N.T., HONG KONG.    
Country: HK
Phone: +852-34262277
Fax: +852-35807610
Email: techsupport@networld.hk
Account Name: HK2803202T




Technical Contact Information:

Family name: LEUNG  
Company name: NETWORLD TECHNOLOGY LIMITED
Address: ROOM 15-18, 16/F., DELTA HOUSE, 3 ON YIU STREET, SHATIN, N.T., HONG KONG.    
Country: HK
Phone: +852-34262277
Fax: +852-35807610
Email: techsupport@networld.hk




Name Servers Information:

NS1.DISCUSS.COM.HK
NS2.DISCUSS.COM.HK



Status Information:

Domain Prohibit Status: 



 -------------------------------------------------------------------------------
 The Registry contains ONLY .com.hk, .net.hk, .edu.hk, .org.hk,
 .gov.hk, idv.hk. and .hk $domains.
 -------------------------------------------------------------------------------

WHOIS Terms of Use 
By using this WHOIS search enquiry service you agree to these terms of use.
The data in HKDNR's WHOIS search engine is for information purposes only and HKDNR does not guarantee the accuracy of the data. The data is provided to assist people to obtain information about the registration record of domain names registered by HKDNR. You agree to use the data for lawful purposes only.

You are not authorised to use high-volume, electronic or automated processes to access, query or harvest data from this WHOIS search enquiry service.

You agree that you will not and will not allow anyone else to:

a.    use the data for mass unsolicited commercial advertising of any sort via any medium including telephone, email or fax; or

b.    enable high volume, automated or electronic processes that apply to HKDNR or its computer systems including the WHOIS search enquiry service; or

c.    without the prior written consent of HKDNR compile, repackage, disseminate, disclose to any third party or use the data for a purpose other than obtaining information about a domain name registration record; or

d.    use such data to derive an economic benefit for yourself.

HKDNR in its sole discretion may terminate your access to the WHOIS search enquiry service (including, without limitation, blocking your IP address) at any time including, without limitation, for excessive use of the WHOIS search enquiry service.

HKDNR may modify these terms of use at any time by publishing the modified terms of use on its website.

