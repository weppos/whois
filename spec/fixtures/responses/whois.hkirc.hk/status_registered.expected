#disclaimer
  %s == "By using this WHOIS search enquiry service you agree to these terms of use.\nThe data in HKDNR's WHOIS search engine is for information purposes only and HKDNR does not guarantee the accuracy of the data. The data is provided to assist people to obtain information about the registration record of domain names registered by HKDNR. You agree to use the data for lawful purposes only.\n"

#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true

#created_on
  %s %CLASS{time}
  %s %TIME{2003-10-14}

#updated_on
  %s %ERROR{AttributeNotSupported}

#expires_on
  %s %CLASS{time}
  %s %TIME{2016-09-23}

#registrar
  %s be_a(Whois::Record::Registrar)
  %s.name == "Hong Kong Domain Name Registration Company Limited"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  %s[0].id            == nil
  %s[0].name          == "NETWORLD TECHNOLOGY LIMITED"
  %s[0].organization  == nil
  %s[0].address       == "ROOM 15-18, 16/F., DELTA HOUSE, 3 ON YIU STREET, SHATIN, N.T., HONG KONG."
  %s[0].city          == "SHATIN",
  %s[0].state         == "N.T."
  %s[0].country       == "HK"
  %s[0].phone         == nil
  %s[0].fax           == nil
  %s[0].email         == "techsupport@networld.hk"
  %s[0].created_on    == Time.parse("2003-10-14")
  %s[0].updated_on    == nil

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "HK2803202T"
  %s[0].name          == "OSCAR LEUNG"
  %s[0].organization  == "NETWORLD TECHNOLOGY LIMITED"
  %s[0].address       == "ROOM 15-18, 16/F., DELTA HOUSE, 3 ON YIU STREET"
  %s[0].city          == "SHATIN",
  %s[0].state         == "N.T."
  %s[0].country       == "HK"
  %s[0].phone         == "+852-34262277"
  %s[0].fax           == "+852-35807610"
  %s[0].email         == "techsupport@networld.hk"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  %s[0].id            == nil
  %s[0].name          == "LEUNG"
  %s[0].organization  == "NETWORLD TECHNOLOGY LIMITED"
  %s[0].address       == "ROOM 15-18, 16/F., DELTA HOUSE, 3 ON YIU STREET"
  %s[0].city          == "SHATIN",
  %s[0].state         == "N.T."
  %s[0].country       == "HK"
  %s[0].phone         == "+852-34262277"
  %s[0].fax           == "+852-35807610"
  %s[0].email         == "techsupport@networld.hk"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.discuss.com.hk"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.discuss.com.hk"

